src/components/application_manager/test/application_helper_test.cc

```diff
@@ -245,7 +245,7 @@ TEST_F(ApplicationHelperTest, RecallApplicationData_ExpectHMICleanupRequests) {
 
   EXPECT_CALL(*mock_message_helper_, SendUnsubscribedWayPoints(_));
 
-  EXPECT_CALL(*mock_message_helper_, SendDeleteCommandRequest(_, _, _));
+  EXPECT_CALL(*mock_message_helper_, CreateDeleteCommandRequests(_, _, _));
```

Comment:
I would say that we should do the ON_CALL for creation, and Expect for rpc_handler.send(

Reply:
Not actual for now